server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4560,22 +4600,20 @@ private UserVmVO commitUserVm(final boolean isImport, final DataCenter zone, fin
             if (s_logger.isDebugEnabled()) {
                 s_logger.debug("Successfully allocated DB entry for " + vm);
             }
-        }
-        CallContext.current().setEventDetails("Vm Id: " + vm.getUuid());
+        } catch (CloudRuntimeException cre) {
+            ArrayList<ExceptionProxyObject> epoList = cre.getIdProxyList();
+            if (epoList == null || !epoList.stream().anyMatch(e -> e.getUuid().equals(vm.getUuid()))) {
+                cre.addProxyObject(vm.getUuid(), "vmId");
```

Comment:
same as above
would be good to use `ApiConstants.VIRTUAL_MACHINE_ID` instead of `"vmid"`